packages/common/src/store/pages/chat/middleware.ts

```diff
@@ -34,13 +29,17 @@ export const chatMiddleware =
             console.debug('[chats] WebSocket opened. Listening for chats...')
           }
           messageListener = ({ chatId, message }) => {
+            const currentUserId = getUserId(store.getState())
```

Comment:
Have a strange feeling that we should be able to get the currentUserId inside the slice and do the comparison there instead of plumbing it down, but maybe not..

Reply:
discussed IRL